tests/basic/src/test/java/org/jboss/hal/testsuite/test/runtime/ServerPreviewTest.java

```diff
@@ -0,0 +1,56 @@
+package org.jboss.hal.testsuite.test.runtime;
+
+import java.io.IOException;
+import java.util.Map;
+
+import org.jboss.arquillian.core.api.annotation.Inject;
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.hal.meta.token.NameTokens;
+import org.jboss.hal.resources.Ids;
+import org.jboss.hal.testsuite.Console;
+import org.jboss.hal.testsuite.Random;
+import org.jboss.hal.testsuite.creaper.ManagementClientProvider;
+import org.jboss.hal.testsuite.fragment.AddResourceDialogFragment;
+import org.jboss.hal.testsuite.fragment.finder.FinderFragment;
+import org.jboss.hal.testsuite.util.ServerEnvironmentUtils;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+
+@RunWith(Arquillian.class)
+public class ServerPreviewTest {
+
+    private static final OnlineManagementClient client = ManagementClientProvider.createOnlineManagementClient();
+    private static final ServerEnvironmentUtils serverEnvironmentUtils = new ServerEnvironmentUtils(client);
+
+    @AfterClass
+    public static void tearDown() throws IOException {
+        client.close();
+    }
+
+    @Drone
+    private WebDriver browser;
+
+    @Inject
+    private Console console;
+
+    @Test
+    public void editUrl() throws IOException {
+        String url = Random.name();
+        FinderFragment fragment = console.finder(NameTokens.RUNTIME);
+        fragment.column(Ids.STANDALONE_SERVER_COLUMN)
+            .selectItem("standalone-host-" + serverEnvironmentUtils.getServerHostName())
+            .dropdown()
+            .click("Edit URL");
+        AddResourceDialogFragment editURLDialog = console.addResourceDialog();
+        editURLDialog.getForm().text("url", url);
+        editURLDialog.add();
+        Map<String, String> mainAttributes = fragment.preview().getMainAttributes();
+        Assert.assertTrue("\"URL\" field is present in Main Attributes", mainAttributes.containsKey("URL"));
+        Assert.assertTrue("\"URL\" field contains value set by test", mainAttributes.get("URL").equals(url));
```

Comment:
Please use `asserEqual` instead of the second `assertTrue` since the possible failure message is more understandable then.